spec/spec_helper.rb

```diff
@@ -70,7 +70,6 @@
     TestServices::delete_es_indexes
     TestServices::create_es_indexes
     require "#{Rails.root}/db/seeds/template.rb"
-    ActiveJob::Base.queue_adapter = :test
```

Comment:
I moved this setting to `config/environments/test.rb` to be consistent with how we set the queue adapter in other environments.